spec/controllers/restaurants_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe RestaurantsController, type: :controller do
```

Comment:
Few things to think about the design here:
1. If Zomato is down then you can't test your code.
2. The seed fetch causes a weird thing where tests are not self contained. index return 2, why? this should be a pard of the test setup and not the seed
3. A single test here tests many things. if it fails it hard to say why.

Things to do:
1. Please move the client to get triggered by a rake task, and do it upon request.
2. on all the tests, run a before flow that adds the restaurants to the response of Restaurant.all request
3. Please break the specs to seperated classes, Client, controller, etc.. and mock the responses.
4. Please use vcr for the api call to zoomato in the specs